react-tic-tac-toe/README.md

```diff
@@ -0,0 +1,14 @@
+# React Tic Tac Toe
+
+A game example built using [TypeScript](https://github.com/Microsoft/TypeScript) and [React](https://github.com/facebook/react), following guidelines from [react-webpack guide](https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/quick-start/react-webpack.md) from TypeScript handbook.
+
+## Build
```

Comment:
Fixed in https://github.com/Microsoft/TypeScriptSamples/pull/86/commits/fbbea5d13b978fd3bfdb462458494be1391000a5